Sage/carbon
# docs(dev): update gpg docs

### Proposed behaviour
Added information how to kill gpg-agent process

### Current behaviour
No information about possible need of restarting gpg-agent.

### Checklist


- [ ] <del>Screenshots are included in the PR if useful
- [ ] <del>Unit tests added or updated if required
- [ ] <del>Cypress automation tests added or updated if required
- [ ] <del>Carbon implementation and Design System documentation are congruent

### Additional context

### Testing instructions
See:
https://github.com/Sage/carbon/blob/dev-docs-update/CONTRIBUTING.md
https://github.com/Sage/carbon/blob/dev-docs-update/docs/dev-environment-setup.md
